### ISSUE TITLE
Skip Navigation link

### ISSUE THREAD

**Issue by erinchambers** [CDRH/african_poetics#395]
Skip to main/skip to navigation link doesn’t appear in the tab order - check if it’s missing or just buggy.

**Comment by erinchambers** [comment]
Site-specific `app/views/layouts/application.html.erb` override was missing the Orchid code for this. Added in the relevant lines of code:

```
<a href="#main-content" class="sr-only sr-only-focusable">
      Skip to main content
</a>
```
